src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -2536,16 +2485,7 @@ void GraphOptimizer::MergeTransposeAndReorder(Graph &graph) {
         auto childChildNode = childNode->getChildEdgeAt(0)->getChild();
 
         auto remEdge = parentNode->getParentEdgesAtPort(1)[0];
-        remEdge->drop();
-        auto& edges = graph.GetEdges();
-        for (auto it = edges.begin(); it != edges.end(); it++) {
-            if ((*it) == remEdge) {
-                edges.erase(it);
-                if (parentParentConstNode->getChildEdges().empty())
-                    parentParentConstNode->remove();
```

Comment:
@luo-cheng2021 @ceciliapeng2011 
Could you please take a look a this particular change.
I didn't get why we need to ->remove() the const input explicitly.
From my understanding edge->drop() is enough in this case.

Reply:
Yes, it should be ok.